app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,45 @@
+class GuestsController < ApplicationController
+  before_action :require_login, only: [:new, :create]
+
+  def show
+
+  end
+
+  def new
+    @guest = Guest.new
+    respond_to do |format|
+      if request.xhr?
+        format.html { render :new, layout: false}
```

Comment:
You could also do 
```
layout = !request.xhr?
format.html { render :new, layout: layout}
```
and ditch the if/else. It might be clearer given that you do the same thing in both branches apart from layout / notlayout